data/helpers.d/filesystem

```diff
@@ -80,3 +80,43 @@ properly with chmod/chown." >&2
     chmod 755 $TMP_DIR
     echo $TMP_DIR
 }
+
+# Determine and store the checkum of a file into tha app config.
+#
+# usage: ynh_store_checksum_config config_file_checksum
+# | arg: config_file_checksum - The file on which the checksum will be perfomed.
+ynh_store_checksum_config () {
+	config_file_checksum=checksum_${1//[\/ ]/_}	# Replace all '/' and ' ' by '_'
+	ynh_app_setting_set $app $config_file_checksum $(sudo md5sum "$1" | cut -d' ' -f1)
+}
+
+# Verify the checksum and backup the file if it's different
+#
+# usage: ynh_compare_checksum_config current_config_file [compress]
+# | arg: current_config_file - The file on which the checksum will be perfomed.
+# | arg: compress - 1 to compress the backup instead of a simple copy
+# A compression is needed for a file which will be analysed even if its name is different.
```

Comment:
+1

Naively I thought the compress option was meant for big files..? I don't understand the "will be analyzed" part.